### ISSUE TITLE
Detroit - Accessibility Features Filters should be OR not AND 

### ISSUE THREAD

**Issue by kathyccheng** [CityOfDetroit/bloom#1218]
Acceptance Criteria 
When I'm filtering by accessibility features, I expect to see properties and that any of the ones listed, not all of them 

**QA NOTES**
https://detroit-public-dev.netlify.app/
select 2 or more accessibility features as filters, the listings should filter properly acting as the accessibility features are OR'd together

**Comment by kathyccheng** [comment]
LGTM
